doc/setup.md

```diff
@@ -181,20 +181,79 @@ To get started using the application with docker,
+
+**NOTE** It's probably a good idea to start the `webpack-dev-sever` to compile webpacks out-of-band of webserver requests - this tends to run into the `rack-timeout` in the docker development environment. The `bin/start` script takes an optional service name - or list of names - to startup. The `webpack-dev-server` _can_ be started issuing the following command:
+
+```
+bin/start webpacker
+```
+
+Additionally, if you don't mind mixing the logs from the `app`, and `webpacker` services you can also startup both like so:
+
+```
+bin/start app webpacker
+```
+
+### Stopping the application and its services
+
+```
+bin/stop
+```
+
+Stops the web application container and its supporting services - RDBMS, mailcatcher, webpacker, etc.
+
+### Obtaining a shell
+
+```
+bin/shell
+```
+
+Starts a shell inside the application container and establishes a TTY connection to it from the user's terminal.
+
+Useful for things like generating migrations, exploring the container file system, etc.
+
+### Deleting all application resources
+
+```
+bin/clean
+```
 
-Note that if you are developing this application, running `make start` a second time after you have made changes may not update the version of the application deployed by `docker-compose`. To ensure that `docker-compose` builds a new image that includes you changes, run `make build` instead.
+Destroys all docker resources for the application and services.
 
-Also, if you would like `docker-compose` to run in daemon mode (which means that it will exit once the images have been set up and the application starts running) you may use `make startd`. This will not show you any logging output from the application, however, and you will not be able to exit the application directly. To view logs when docker-compose is running in daemon mode, use `make logs`. To stop the application and all its services, run `make stop`.
+### Notes
 
-**NOTE** the application will save its state between successive invocations of `make build`. This means that if you make changes to the database - for example by adding content or users - then those changes will persist the next time you start the application with `make start`. You can wipe all the state of the application and all the services (including the postgres database) attached to it by running `make wipe`. In particular, you may need to do this if you are making breaking changes to the database structure, or if you have corrupted something somehow. However, do be careful, because this will delete **all** the state saved in the application and database - and there is no way to retrieve it. So make sure you back up anything you want to save before running the command.
+- Of course, all of the above binstubs are simply short hands for various `docker-compose` incantations - there's nothing stopping your from simply using `docker-compose` to interact with the services listed in the [`docker/development/docker-compose.yml`](../docker/development/docker-compose.yml).
```

Comment:
stopping 'you' ?

Reply:
Resolved by https://github.com/rubyforgood/mutual-aid/pull/715/commits/b10bebb225c7cba71319ccdf3dbaf97c87dcf74d